tests/restarting/from_6.2.0/SnapCycleRestart-2.txt

```diff
@@ -1,8 +1,10 @@
 testTitle=SnapCycleRestore
 ;Post snap restore test
-        runSetup=false
-        testName=Cycle
-        transactionsPerSecond=2500.0
-        nodeCount=2500
-        testDuration=10.0
-        expectedRate=0
+runSetup=false
+testName=Cycle
+transactionsPerSecond=2500.0
+nodeCount=2500
+testDuration=10.0
+expectedRate=0
+buggify=off
```

Comment:
Do we have a policy here? Something like the feature is considered experimental until the tests work with `buggify` on?

Reply:
This is a special case of why we do buggify off here, buggifying restore asks for more changes to regular recovery which is only needed during restore time. 
Also, if a tlog fails during restore recovery, it can be retried, so with the idea of keeping the logic simple and the fact that we can retry a restore it was decided to go with buggify off.